packages/react/src/CTAForm/CTAForm.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import clsx from 'clsx'
+import {FormControl, Button, ButtonProps, TextInput, TextInputProps, Checkbox} from '../'
+
+import styles from './CTAForm.module.css'
+
+type CTAFormProps = {
+  inputLabel: TextInputProps['leadingText']
+  inputType?: TextInputProps['type']
+  placeholder?: TextInputProps['placeholder']
+  buttonLabel: ButtonProps<'button'>['children']
+  checkboxLabel: React.ReactNode[] | React.ReactNode | string
+} & React.HTMLAttributes<HTMLFormElement>
+
+export const CTAForm = ({
+  inputLabel,
+  inputType = 'text',
+  placeholder,
+  buttonLabel,
+  checkboxLabel,
+  ...args
```

Comment:
is `args` a storybook hangover? Can we change to `rest` here?

```suggestion
  ...rest
```

Reply:
in next push